core/src/test/java/com/graphhopper/reader/dem/BridgeElevationInterpolatorTest.java

```diff
@@ -0,0 +1,133 @@
+package com.graphhopper.reader.dem;
+
+import static org.junit.Assert.assertEquals;
+
+import java.util.Arrays;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import com.graphhopper.coll.GHBitSetImpl;
+import com.graphhopper.reader.ReaderWay;
+import com.graphhopper.routing.util.DataFlagEncoder;
+import com.graphhopper.routing.util.EncodingManager;
+import com.graphhopper.storage.GraphExtension;
+import com.graphhopper.storage.GraphHopperStorage;
+import com.graphhopper.storage.NodeAccess;
+import com.graphhopper.storage.RAMDirectory;
+import com.graphhopper.util.EdgeIteratorState;
+import com.graphhopper.util.Helper;
+
+import gnu.trove.set.hash.TIntHashSet;
+
+public class BridgeElevationInterpolatorTest {
+
+	private static final double EPSILON= 0.0000000001d;
+	private static final ReaderWay BRIDGE_WAY;
```

Comment:
I do not like mutable static variables that much - you do not write to it but it is allowed and could then introduce problems for multithreaded unit test execution. In test we also do not need to care about object creation so I'd vote to just remove the static and put the initialization in the `@Before` method.